go/vt/vtctl/workflow/utils.go

```diff
@@ -1019,3 +1021,59 @@ func applyTargetShards(ts *trafficSwitcher, targetShards []string) error {
 	}
 	return nil
 }
+
+// validateEmptyTables checks if all specified tables are empty across specified shards.
+// It queries each shard's primary tablet and if any non-empty table is found, it returns an error
+// containing a list of non-empty tables.
+func validateEmptyTables(ctx context.Context, ts *topo.Server, tmc tmclient.TabletManagerClient, shards []*topo.ShardInfo, tableSettings []*vtctldatapb.TableMaterializeSettings) error {
+	var mu sync.Mutex
+	isTableFaulty := map[string]bool{}
```

Comment:
Change terminology from "Faulty" to "NonEmpty" like `hasTableData` or `isNonEmpty`. Similarly `faultyTables` => `nonEmptyTables` or `tablesWithData` ...

Reply:
Done.